doc/configuration.md

```diff
@@ -0,0 +1,55 @@
+# How To configure Tapyrus Signer Network
+
+This document describes how to configure tapyrus-signer.
+
+After setting up with [How to set up new Tapyrus Signer Network](./setup.md), We can start Tapyrus Signer Network.
+
+We use some parameters introduced in setup documentation.
+
+- `node_private_key`: A private key on the secp256k1 elliptic curve.
+- `node_public_key` : A public key corresponding to `node_private_key`.
+- `aggregated_public_key`: An aggregated public key generated by using setup command `tapyrus-setup aggregate`
+- `node_secret_share`: A secret value generated by using setup command `tapyrus-setup aggregate`
+
+## Configuration
+
+To configure for signer node, we use signer_config.toml.
+
+The following are mandatory settings to start nodes.
+For all settings including optional ones, see [sample file](https://github.com/chaintope/tapyrus-signer/blob/master/tests/resources/signer_config_sample.toml)
+
+### [general] section
+
+`[general]` section is a set of settings for application.
+
+- There is no mondatory settings in this section.
+
+### [signer] section
+
+`[signer]` section is a set of settings for consensus algorithm used in Tapyrus Signer/Core Network.
+
+- `publickeys`: Public keys of all signers that participate this Tapyrus Signer Network. Specify in a compressed public key format.
+- `privatekey`: Private key of this signer node. Specify in an extend WIF format. Use same key with `node_private_key` generated in setting up.
+- `threshold`: The number of signers required to sign block. It should be less than the number of all signers.
+- `aggregated_public_key`: Aggregated public key generated in setting up.
+- `node_secret_share`: Secret value generated in setting up.
```

Comment:
I think we should make `[signer]` section possible to take commitments for vss. Is it right? The verification local signature should use it.

Reply:
Fix at ffc673e712e22941e89fee0a90199a9d925b4724